### PR TITLE
Fix/manifest start url

### LINKED ISSUE #299: Fix manifest start url
Currently the `start_url` from the manifest served is set to: `https://{community_subdomain}.cambiatus.io/api/{community_subdomain}.cambiatus.io`. Per the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Add_to_home_screen) the `start_url`:

> Provides a path to the asset that should be loaded when the added-to-Home screen app is launched. Note that this has to be a relative URL pointing to the site index, relative to the URL of the manifest.

Therefore, the `start_url` should point to `https://{community_subdomain}.cambiatus.io/`

Bug reported by @henriquecbuss

### PR DESCRIPTION
**Pull request by MatheusBuss**
## What issue does this PR close
Closes #299 

## Changes Proposed ( a list of new changes introduced by this PR)
Change `start_url` on the manifest served to the root of the website

### CODE REVIEW

File: lib/cambiatus_web/templates/email/claim.html.eex [hunk 1]
```diff
@@ -204,7 +204,7 @@
                                           </b>
                                         </div>
                                         <div style="text-align: center"><br></div>
-                                        <div style="text-align: center"> <%= gettext("Your claim was approved!") %>
+                                        <div style="text-align: center"> <%= gettext("Your claim on the action") %><span style="font-weight:bold;"><%= " #{@claim.action.description} " %></span><%= gettext("was approved!") %>
```
Comment (henriquecbuss) [inline lib/cambiatus_web/templates/email/claim.html.eex:207]: Is this intended? 🤔

**Review by henriquecbuss — APPROVED** [review]
Working on staging 👍

File: lib/cambiatus_web/templates/email/claim.html.eex (hunk 1 above)
Comment (MatheusBuss) [reply lib/cambiatus_web/templates/email/claim.html.eex:207]: I must've had a commit that i hadn't pushed on another branch before :sweat_smile:

Comment (henriquecbuss) [reply lib/cambiatus_web/templates/email/claim.html.eex:207]: From 5 September 😆 
image

**Review by lucca65 — APPROVED** [review]

**Pull request merged**